server/service/handler.go

```diff
@@ -506,6 +503,15 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	// Fleet Sandbox demo login (always errors unless config.server.sandbox_enabled is set)
 	ne.WithCustomMiddleware(limiter.Limit("login", throttled.RateQuota{MaxRate: loginRateLimit, MaxBurst: 9})).
 		POST("/api/_version_/fleet/demologin", makeDemologinEndpoint(config.Server.URLPrefix), demologinRequest{})
+
+	ne.WithCustomMiddleware(
```

Comment:
Thinking out loud, will we ever want an Orbit to know Desktop capabilities of the server?
If so, having only one ping endpoint and returning all (orbit + desktop) capabilities together would simplify that scenario.

> Thinking out loud, will we ever want an Orbit to know Desktop capabilities of the server?

Answering myself, I guess that if we want this then we add such Desktop capability to the Orbit capability set. :)

Reply:
woot, that sounds good to me